### ISSUE TITLE
Test should be skipped if ipykernel not installed

### ISSUE THREAD

**Issue by aaronayres35** [enthought/envisage#369]
When running the test suite in an environment in which `ipykernel` was not installed the following test failure occurs:
```
======================================================================
ERROR: tests.test_ids (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.test_ids
Traceback (most recent call last):
  File "/Users/aayres/.edm/envs/test-envisage/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/Users/aayres/.edm/envs/test-envisage/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/Users/aayres/Desktop/envisage/envisage/tests/test_ids.py", line 15, in <module>
    from envisage.plugins.ipython_kernel.api import IPythonKernelPlugin
  File "/Users/aayres/Desktop/envisage/envisage/plugins/ipython_kernel/api.py", line 17, in <module>
    from envisage.plugins.ipython_kernel.actions import StartQtConsoleAction
  File "/Users/aayres/Desktop/envisage/envisage/plugins/ipython_kernel/actions.py", line 13, in <module>
    from .internal_ipkernel import InternalIPKernel
  File "/Users/aayres/Desktop/envisage/envisage/plugins/ipython_kernel/internal_ipkernel.py", line 20, in <module>
    import ipykernel.connect
ModuleNotFoundError: No module named 'ipykernel'


----------------------------------------------------------------------
```

This test should simply be skipped if `ipykernel` is not installed

**Comment by bnavigator** [comment]
closed by #423

**Comment by mdickinson** [comment]
Thanks; closing. (It looks as though I failed to notice that #420 was a duplicate of this issue.)
